frontend/src/pages/UserProfileConfiguration/ChangeUserPassword.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect, ChangeEvent } from 'react';
+import { CheckUserAuth } from 'pages/CheckUserAuth';
+import { Navbar } from '../../components/Navbar';
+import { useCookies } from 'react-cookie';
+import { ChangeUserPassword } from '../../Utils/Utils';
+
+function ChangeUserEmail() {
+  const [token] = useCookies(['mytoken']);
+  const [userCookie] = useCookies(['user']);
+  const tokenId = token.mytoken;
+  const userId = userCookie.user.id;
+  const [userPasswordChangeConfiguration, setuserPasswordChangeConfiguration] = useState<{
+    [key: string]: string;
+  }>({
+    new_password: '',
+    re_new_password: '',
+    current_password: ''
+  });
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) =>
+    setuserPasswordChangeConfiguration({
+      ...userPasswordChangeConfiguration,
+      [event.target.name]: event.target.value
+    });
+
+  const changePasswordFormConfiguration = [
+    { name: 'new_password', id: 'new_password', placeholder: 'New Password', type: 'password' },
+    {
+      name: 're_new_password',
+      id: 're_new_password',
+      placeholder: 'Confirm New Password',
+      type: 'password'
+    },
+    {
+      name: 'current_password',
```

Comment:
I think that current password should be first input, from user experience it might be better approach